src/utils/post_utils.ts

```diff
@@ -37,7 +37,10 @@ export function isPostEphemeral(post: Post): boolean {
     return post.type === Posts.POST_TYPES.EPHEMERAL || post.type === Posts.POST_TYPES.EPHEMERAL_ADD_TO_CHANNEL || post.state === Posts.POST_DELETED;
 }
 
-export function shouldIgnorePost(post: Post): boolean {
+export function shouldIgnorePost(post: Post, userId?: $ID<UserProfile>): boolean {
+    if ((post.type && post.type === Posts.POST_TYPES.ADD_TO_CHANNEL) && (post.props.addedUserId && post.props.addedUserId === userId)) {
```

Comment:
Can we add tests for the exception of `ADD_TO_CHANNEL` type here?

Reply:
Yep, I'll add that.